docs/coding-conventions.md

```diff
@@ -320,6 +320,98 @@ coding conventions](#detailed-coding-conventions).
       subdirectories.
 - Document directories and filenames should use dashes rather than underscores.
 
+# Writing Tests
+## Writing functional tests
+### Test Interrupts
+To make sure that any asynchronous operation is canceled when the test is
+terminated, make sure to use golang contexts properly. See more details in 
```

Comment:
This is more of a combination of ginkgo and golang, not just golang contexts. I think you could explain the underlying problem a bit better. What does it mean to use contexts properly?

Reply:
Not sure I need to quote ginkgo documentation, if I place the link to it right there.